src/Nethermind/Nethermind.Merge.Plugin/Metrics.cs

```diff
@@ -34,10 +34,22 @@ public static class Metrics
 
         [GaugeMetric]
         [Description("Number of responses to engine_getBlobsV1 and engine_getBlobsV2 with all requested blobs")]
-        public static int NumberOfGetBlobsSuccesses { get; set; }
+        public static int GetBlobsRequestsSuccessTotal { get; set; }
 
         [GaugeMetric]
         [Description("Number of responses to engine_getBlobsV1 and engine_getBlobsV2 without all requested blobs")]
-        public static int NumberOfGetBlobsFailures { get; set; }
+        public static int GetBlobsRequestsFailureTotal { get; set; }
+
+        [CounterMetric]
+        [Description("Number of Blobs requested by engine_getBlobsV2")]
+        public static int GetBlobsRequestsTotal { get; set; }
+
+        [CounterMetric]
+        [Description("Number of Blobs requested by engine_getBlobsV2 that are present in the blobpool")]
+        public static int GetBlobsRequestsInBlobpoolTotal { get; set; }
```

Comment:
We have similar metrics already - `GetBlobsRequestsTotal` is like `NumberOfRequestedBlobs` and `GetBlobsRequestsInBlobpoolTotal` is like `NumberOfSentBlobs`.
But I think it is reasonable to introduce new, standardized metrics for `engine_getBlobsV2`, while keep using old ones only for `engine_getBlobsV1`